schema/deploy/shipping/views.sql

```diff
@@ -139,26 +139,37 @@ create or replace view shipping.return_results_v1 as
              when count(present) > 0 then 'complete'
            end as status,
            -- We only return the top level organisms for results so we want to omit subtypes
-           array_agg(distinct subpath(organism, 0, 1)::text)
-            filter (where present and organism is not null)  as organisms_present,
-           array_agg(distinct subpath(organism, 0, 1)::text)
-            filter (where not present and organism is not null) as organisms_absent
+           array_agg(distinct
+             (case
+              when 'Influenza'::ltree @> organism then subpath(organism, 0, 2)::text
+              else subpath(organism, 0, 1)::text
+             end)
+           ) filter (where present and organism is not null)  as organisms_present,
+           array_agg(distinct
+             (case
+              when 'Influenza'::ltree @> organism then subpath(organism, 0, 2)::text
+              else subpath(organism, 0, 1)::text
+             end)
+           ) filter (where not present and organism is not null)  as organisms_absent
```

Comment:
The `subpath` stuff isn't needed and this can be much simpler since we're only selecting results for `Influenza.A`, `Influenza.B`, and `RSV` now: `array_agg(distinct organism::text)`.

Reply:
I thought it would still include the subtypes of `Influenza.A` and `Influenza.B` without the subpath stuff. 

Although all Cepheid results don't include subtypes, so it might be overly complicated.

Reply:
Ahh, yeah, i was thinking only about the Cepheid results, but now recall we do have those subtypes.

You're definitely correct that the code will still guard against Cepheid being more specific (or us otherwise introducing results that are more specific). I'm not sure that'll ever happen, but no problem being defensive. No need to change this then.

Reply:
I can also change to the where clause to `organism ? '{"Influenza.A", "Influenza.B", "RSV"}'::lquery[]` for exact matches only

Reply:
Nod. I'm ok with either way.